R/project-element.R

```diff
@@ -0,0 +1,124 @@
+#' @title Add project element 
+#' @description This function creates a project node within the parent element that 
+#' contains all the required elements for the project section of an EML document.
+#' This function can be used in combination with add_personnel
+#' and add_funding. Add_personnel can be used to generate the project_personnel 
+#' input and add_funding can be used to generate the award_information input.  
+#' @param parent_element A list in which the project node should be nested in. 
+#' @param project_title The title of the project that the funding is awarded to. 
+#' @param award_information A list that includes the required funding information 
+#' for an EML document.This list must include the award title and the funderName.
+#' This list can be created by calling the add_funding function 
+#' on the funding information or by manually inputting the required information. 
+#' If the list is written manually it must be formatted as follows. 
+#'
+#' award_infomation = list(funderName = "Name", title = "Award Title")
+#'
+#' Additional information about the funding may be added to the list. See the 
+#' add_funding documentation for more information. 
```

Comment:
make this a link to that function documentation `\code{\link{add_funding}}`

Reply:
fixed